### ISSUE TITLE
add an example of TWEAK

### ISSUE THREAD

**Issue by tbrowder** [Raku/doc#1796]
also explain its advantages over other methods of object contruction

**Comment by JJ** [comment]
Where? For instance, here: https://docs.perl6.org/language/objects ? Right now there is this example:

```
class RectangleWithCachedArea {
    has ($.x1, $.x2, $.y1, $.y2);
    has $.area;
    submethod TWEAK() {
        $!area = abs( ($!x2 - $!x1) * ( $!y2 - $!y1) );
    }
}
 
say RectangleWithCachedArea.new( x2 => 5, x1 => 1, y2 => 1, y1 => 0).area;
```
It's also mentioned several times. I don't think it really deals with its advantages...

**Comment by tbrowder** [comment]
I didn't see the example because it is far below the search  reference point for TWEAK. I added a note there that there is an example later in the text.  Closed with  1b723a26bc4ea3c09a983f5e351e697860c56da8.

**Comment by JJ** [comment]
Reopen with slightly changed title.

**Comment by tbrowder** [comment]
This is still troubling me for the paucity of a real-world example. I asked for help for a practical class construction on #raku a couple of days ago and got some info that is helpful but I don't find it in the docs. I'm looking for the discussion reference now and will add it here.

**Comment by JJ** [comment]
Please do.

**Comment by tbrowder** [comment]
Wrong, TWEAK enables me to do exactly what I need for my use case.

**Comment by tbrowder** [comment]
Submethod TWEAK is missing on the Type system page (typesystem.pod6)
